lib/modules/filteReddit/Filterline.js

```diff
@@ -409,6 +409,41 @@ export default class Filterline {
 		this.refreshThing(thing);
 	}
 
+	checkEmptyState = (() => {
+		let notification;
+		let showId;
+
+		const showNotification = () => {
+			const info = $('<p>').text(i18n('filteRedditEmptyNotificationInfo'))
+			const toggle = $('<button>').text(i18n('filteRedditEmptyNotificationToggleShowReason'))
+				.click(() => { this.toggleShowFilterReason(); });
+
+			notification = Notifications.showNotification({
+				moduleID: FilteReddit.module.moduleID,
+				notificationID: 'everyThingHidden',
+				header: i18n('filteRedditEmptyNotificationHeader'),
+				message: $('<div>').append(info).append(toggle).get(0),
+				closeDelay: Infinity,
+			});
+		};
+
+		return _.throttle(() => {
+			if (this.getFiltered().length !== this.things.size) {
+				if (Number.isInteger(showId)) clearTimeout(showId);
```

Comment:
Could use `debouncedFunction.cancel()` for this to avoid tracking the timeout id